### PR TITLE
CUMULUS-254 Add README and examples

### PR DESCRIPTION
**Pull request by yjpa7145**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,92 @@
...
+
+## Why?
+
+This approach has a few major advantages:
+
+1. It explicitly prevents tasks from making assumptions about data structures
+   like `meta` and `cumulus_meta` that are owned internally and may therefore
+   be broken in future updates. To gain access to fields in these structures,
+   tasks must be passed the data explicitly in the workflow configuration.
+1. It provides clearer ownership of the various data structures. Operators own
```
Comment (laurenfrederick) [inline README.md]: These are all 1.

File: README.md [hunk 2]
```diff
@@ -1 +1,92 @@
-# cumulus-message-adapter-js
\ No newline at end of file
+# @cumulus/cumulus-message-adapter-js
```
Comment (laurenfrederick) [inline README.md]: Maybe add something about how this needs to be packaged for lambda

File: README.md (hunk 1 above)
Comment (yjpa7145) [reply README.md]: That's just Markdown syntax for a numbered list.  When it's rendered they'll have the correct numbers.

File: README.md (hunk 2 above)
Comment (yjpa7145) [reply README.md]: It seems like teaching people how to create a Lambda package is out of scope for this, which is just about how to use the message adapter.  If there's some wiki page on how to do that then I can link to it.  @scisco, is there any page like that out there?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+Example of a good commit message;
+
+```
+Improve contributing guidelines. Fixes #10
+
+Improve contributing docs and consolidate them in the standard location https://help.github.com/articles/setting-guidelines-for-repository-contributors/
+```
+
+### For more information on Cumulus governance, see the [Cumulus Code Contribution Guidelines](https://docs.google.com/document/d/14J_DS6nyQ32BpeVjdR-YKfzHAzFB299tKghPGshXUTU/edit).
```
Comment (laurenfrederick) [inline CONTRIBUTING.md]: Can you add "and [the Cumulus Wiki](https://wiki.earthdata.nasa.gov/display/CUMULUS/Cumulus)." to the end of the last sentence? See CUMULUS-292.

File: README.md (hunk 2 above)
Comment (laurenfrederick) [reply README.md]: Talked to @scisco earlier today and he said to put it in the README. But up to you. I am going to work on a better format for this.

File: README.md (hunk 1 above)
Comment (laurenfrederick) [reply README.md]: Got it. Thanks!

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Submitting an Issue
+
+If you have any questions or ideas, or notice any problems or bugs, first [search open issues](https://github.com/cumulus-nasa/cumulus-message-adapter-js/issues) to see if the issue has already been submitted. We may already be working on the issue. If you think your issue is new, you're welcome to [create a new issue](https://github.com/cumulus-nasa/cumulus-message-adapter-js/issues/new).
+
+## Pull Requests
+
+If you want to submit your own contributions, follow these steps;
+
+* Fork the Cumulus repo
```
Comment (laurenfrederick) [inline CONTRIBUTING.md]: Change this to "this repo" or the name of this repo.

File: README.md (hunk 2 above)
Comment (yjpa7145) [reply README.md]: Fixed.

File: CONTRIBUTING.md (hunk 4 above)
Comment (yjpa7145) [reply CONTRIBUTING.md]: Fixed.

File: CONTRIBUTING.md (hunk 3 above)
Comment (yjpa7145) [reply CONTRIBUTING.md]: Fixed.

**Pull request merged**